CONTRIBUTING.md

```diff
@@ -67,28 +67,28 @@ Currently, this project does not have a consistent style for Markdown files nor
 
 ## Testing locally
 
-This project uses [MkDocs](https://www.mkdocs.org/) tool with [Material theme](https://squidfunk.github.io/mkdocs-material/) to generate the website.
+This project uses [MkDocs](https://www.mkdocs.org/) tool with [Material theme](https://squidfunk.github.io/mkdocs-material/) and extra plugins to generate the website.
 
-- To test locally, you will need to install Material for MkDocs:
-```
-pip install mkdocs-material
-```
-
-- Once done, you need to run MkDocs. Simply type the following commands:
+- To test locally, you will need to install the dependencies. To do that, type the following commands:
 ```
 git clone https://github.com/learn-neuroimaging/tutorials-and-resources.git
 cd tutorials-and-resources
-mkdocs serve
+pip install -r requirements.txt
```

Comment:
you think you can rename it into pip-requirements.txt ? :pleading_face:

Reply:
See 1ce0830 :)